solidity/dashboard/src/components/LiquidityRewardCard.jsx

```diff
@@ -65,14 +66,45 @@ const LiquidityRewardCard = ({
           className="text-small"
         >
           View pool
-        </a>
+        </a>&nbsp;
+        <Tooltip
+          simple
+          delay={0}
+          triggerComponent={Icons.MoreInfo}
+          className={"liquidity__card-subtitle__tooltip"}
+        >
+          LP tokens represent the amount of money you've deposited into a
+          liquidity pool as a liquidity provider. KEEP rewards are proportional
+          to your share of the total pool.
+        </Tooltip>
       </h4>
-      <div className={"liquidity__info text-grey-60 mt-2"}>
+      <div className={"liquidity__info text-grey-60 mt-2 mb-2"}>
         <div className={"liquidity__info-tile bg-mint-10"}>
+          <Tooltip
+            simple
+            delay={0}
+            triggerComponent={Icons.MoreInfo}
+            className={"liquidity__info-tile__tooltip"}
+          >
+            Pool APY is calculated using the&nbsp;
+            <a
+              target="_blank"
+              rel="noopener noreferrer"
+              href={
+                "https://thegraph.com/explorer/subgraph/uniswap/uniswap-v2"
+              }
+              className="text-small text-white text-link"
```

Comment:
Probably `text-small` and `text-white` are not needed here.

Reply:
I think we need `text-white` since the links aren't white by default but i removed `text-small` 👍